LDM-732.tex

```diff
@@ -60,35 +55,22 @@ \section{Introduction}\label{sec:intro}
 
 \subsection{Scope}\label{sec:scope}
 
-This plan governs only tests of the network infrastructure, not the applications and services that
-use the network. To be specific, this plan governs tests of the network only up to ISO OSI Layer
-\url{https://en.wikipedia.org/wiki/OSI_model}.
+This document pertains only to the verification of the network infrastructure, not the applications and services that use the network, specifically, up to \href{https://en.wikipedia.org/wiki/OSI_model}{OSI Layer} 3.
 
-As such, all of the tests governed by this plan and defined in the
-\href{https://jira.lsstcorp.org/projects/LVV/}{LSST Verification and Validation JIRA Project (LVV)}
-are defined as Lower Level (LL) in the System Engineering test hierarchy. LL
-corresponds to Data Management Subsystem Integration. Where appropriate, additional
-comments regarding Same Level (SL) which corresponds to LSST System Integration, and Higher
-Level (HL) which corresponds to LSST Commissioning, are called out in the Verification
-Elements.
+As such, all of the Verification Elements governed by this document and defined in the \href{https://jira.lsstcorp.org/projects/LVV/}{LSST Verification and Validation (LVV)} Jira project are Lower Level (LL) in the System Engineering test hierarchy.
```

Comment:
@leannep Does the project consider this document or Jira LVV the ultimate source of truth?

Reply:
This document should be identical to what is in Jira (at least at the time the document was generated) , modulo some additional narrative.  Jira LVV can clearly evolve.  Perhaps change to:
As such, all of the Verification Elements defined in the \href{https://jira.lsstcorp.org/projects/LVV/}{LSST Verification and Validation (LVV)} Jira project and presented in this document are Lower Level (LL) in the System Engineering test hierarchy.

Reply:
jira is the source of truth, these documents provide a snapshot aat a given time, usually associated with a intermediate milestone.

Reply:
Thank you for clarifying. I've incorporated the suggest re-wording.